exampleCourse/questions/demo/autograder/java/helloWorld/info.json

```diff
@@ -9,6 +9,7 @@
   "externalGradingOptions": {
     "enabled": true,
     "image": "prairielearn/grader-java",
-    "timeout": 10
+    "timeout": 10,
+    "environment": { "JDK_JAVAC_OPTIONS": "--release 11" }
```

Comment:
Added to allow testing of environment variables and to ensure JUnit still works with older versions of Java (since JUnit 6 requires Java 17 or higher for execution).